src/chains/ethereum/ethereum/tests/api/eth/sign.test.ts

```diff
@@ -40,14 +41,11 @@ describe("api", () => {
           address,
           Data.toString(msg)
         ]);
-        sgn = sgn.slice(2);
+        const { v, r, s } = fromRpcSig(sgn);
 
-        const r = Buffer.from(sgn.slice(0, 64), "hex");
-        const s = Buffer.from(sgn.slice(64, 128), "hex");
-        const v = parseInt(sgn.slice(128), 16);
         const pub = ecrecover(msgHash, v, r, s);
         const addr = fromSigned(pubToAddress(pub));
-        const strAddr = "0x" + addr.toString("hex");
+        const strAddr = Data.toString(Quantity.toBuffer(addr), 20);
```

Comment:
That's a bit of a pain that Data doesn't accept the bigint arg, especially now that the type is being used more widely.

Maybe we should support bigint in Data ctor?

In the meantime, I was initially confused by why we were doing this - given it's a test, would it be clearer to do this?
```suggestion
        const strAddr = Data.toString(`0x${addr.toString(16)}`, 20);
```